### PR TITLE
Refactor Box/Tools to remove unnecessary functions

### PR DESCRIPTION
**Pull request by admdly**
Refactor Box/Tools to remove functions which essentially duplicate PHP functionality, or provide a thin wrapper around such functions.

This is a relatively tame refactoring, and no doubt it could go much further, but none of the changes should be breaking. 🙂

### CODE REVIEW

File: src/modules/Spamchecker/Service.php [hunk 1]
```diff
@@ -133,7 +133,7 @@ public function isInStopForumSpamDatabase(array $data)
     {
         $data['f'] = 'json';
         $url = 'https://www.stopforumspam.com/api?' . http_build_query($data);
-        $file_contents = $this->di['tools']->file_get_contents($url);
+        $file_contents = file_get_contents($url);
```
Comment (jaapmarcus) [inline src/modules/Spamchecker/Service.php:136]: I thought the plan was rewriting all file_get_contents to http module?

**Review by jaapmarcus — APPROVED** [review]

File: src/modules/Spamchecker/Service.php (hunk 1 above)
Comment (jaapmarcus) [reply src/modules/Spamchecker/Service.php:136]: https://github.com/FOSSBilling/FOSSBilling/pull/861

Comment (admdly) [reply src/modules/Spamchecker/Service.php:136]: 100% the plan is long-term, however the reality is these functions will likely not survive 1-2months so I avoided doing too much at once. We need a much better spam handling module, integrating up to date CAPTCHA, etc.

**Review by BelleNottelling — APPROVED** [review]
LGTM, glad to get this cleaned up

**Pull request merged**